backend/migrations/20201026102610_alter-table-completion-add-tier.ts

```diff
@@ -0,0 +1,13 @@
+import * as Knex from "knex"
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`ALTER TABLE completion
+    ADD COLUMN IF NOT EXISTS tier INTEGER;
```

Comment:
Since we already have completions for building ai should we somehow populate the tier to those completions? We can either do it in the migration or do it manually in the db.

Reply:
Yes, probably. Some script that runs `getBAIProgress` to get the highest tier and then `createCompletion` to update the tier in the completions.